pontoon/base/models.py

```diff
@@ -338,7 +342,9 @@ def user_locale_role(self, locale):
 @property
 def contributed_translations(self):
     """Filtered contributions provided by user."""
-    return Translation.objects.filter(user=self)
+    return Translation.objects.filter(
+        user=self,
+        entity__resource__project__is_dummy=False)
```

Comment:
Please put closing parentheses on a new line, and add a comma to the end of this line (to make following diffs smaller).

Reply:
:+1: